source/js/common/initialize-primary-nav.js

```diff
@@ -7,6 +7,8 @@ import navNewsletter from "../nav-newsletter.js";
  * @param {Object} primaryNavModule primary nav module to initiate
  */
 export const initializePrimaryNav = (siteUrl, csrfToken, primaryNavModule) => {
-  primaryNavModule.init();
+  if (primaryNavModule && typeof primaryNavModule.init === `function`) {
```

Comment:
We should be able to assume that if `primaryNavModule` exists, it's the correct shape and `primaryNavModule.init()` will work. If not, we probably have a problem elsewhere (as in, we're calling initializePrimaryNav with something in the spot for `primaryNavModule` that isn't actuallly a `primaryNavModule`...)

Reply:
Alright. Will update!